release-notes/search-processor.release-notes-2.6.0.md

```diff
@@ -0,0 +1,13 @@
+## Version 2.6.0 Release Notes
+
+### Enhancements
+* Add more test coverage [#105](https://github.com/opensearch-project/search-processor/pull/105)
```

Comment:
Use PR from main branch https://github.com/opensearch-project/search-processor/pull/100

Same for the rest